xml/art_slert_hw_testing.xml

```diff
@@ -176,95 +176,95 @@
     <title>Determining Latencies with the Hardware Latency Detector</title>
 
     <para> The <emphasis>hardware latency detector</emphasis>
-        (<filename>hwlat_detector.ko</filename>) is installed with the
-      real-time kernel (the <package>kernel-rt</package> package).
+        (<command>hwlatdetect</command>) is a program that controls the kernel
+      latency detector module (hwlat_detector.ko), installed with the
+      real-time debug kernel (the <package>kernel-rt_debug</package> package).
       It is a special kernel module that is used to detect large system
       latencies. It is caused by the behavior of underlying hardware or
       firmware, independent of Linux itself (system management
       interrupts, SMIs). </para>
 
     <para> The hardware latency detector module works by making all CPUs
       busy for a configurable amounts of time. This is done by calling
-      <systemitem class="other" otherclass="function"
-       >stop_machine()</systemitem>, polling the CPU Time Stamp Counter (TSC)
+      <systemitem class="other" otherclass="function">stop_machine()</systemitem>, polling the CPU Time Stamp Counter (TSC)
       for some period, and then looking for gaps in the TSC data.
       These gaps indicate times when the polling was disturbed by
       System Management Interrupts (SMI).</para>
 
     <para> To start using hardware latency detector, use as
-      &rootuser;: </para>
+      <systemitem class="username">root</systemitem>: </para>
 
-    <screen>modprobe hwlat_detector enabled=1 threshold=10</screen>
+    <screen>hwlatdetect threshold=10</screen>
 
     <para> The <literal>threshold</literal> value is the minimum latency
-      value to be considered in microseconds. Use the value that is
-      relevant for your particular situation. </para>
+      value to be considered in microseconds. Use the value which
+      is relevant for your particular situation. </para>
 
-    <para> After the module is loaded, the
-        <filename>/sys/kernel/debug/hwlat_detector/</filename> directory
-      is created. The directory contains several files: </para>
+    <para>
+        <command>hwlatdetect</command> has the below options: </para>
 
     <variablelist>
       <varlistentry>
-        <term><filename>count</filename></term>
+        <term><filename>--duration="time"{s,m,d}</filename></term>
+        <listitem>
+          <para> Run the detector logic in for the specified duration. The duration is a base 10
+              integer number that defaults to a value in seconds. An optional suffix may be
+              specified to indicate minutes, hours or days. </para>
```

Comment:
The sentence "The duration is a base 10 integer number that defaults to a value in seconds" seems clumsy.

How about this?

"Provide the duration as a decimal integer. The default unit is seconds. To specify minutes, hours, or days, append the initial of the desired units. The units are case-sensitive."

(Case sensitive or insensitive as appropriate, obviously.)